scripts/build_mod_ic.py

```diff
@@ -401,6 +402,21 @@
         "chip_shape": (3.1, 4.1),
         "pin_shape": (-0.41, 0.28)
     },
+    
+    # HVQFN24 from PCA9502
+    # IPC-7351B: QFN50P430X430X100-25N
+    "HVQFN24-NXP": {
+        "rows": 4,
+        "pins": 24,
+        "pin_pitch": 0.5,
+        "row_pitch": 4.10,
+        "pad_shape": (0.90, 0.24),
+        "ep_shape": (2.10, 2.10),
+        "ep_mask_shape": (0.45, 0.45, 0.4, 0.4),
+        "ep_paste_shape": (0.45, 0.45, 0.4, 0.4),
+        "chip_shape": (4.3, 4.3),
+        "pin_shape": (0.475, 0.45)
```

Comment:
Ah, I actually meant the `pin_shape` property (which is just the Fab layer drawing of how wide the pin is). Thinking about it you should have a negative x-dimension too as the pins are internal. If you turn on the F.Fab layer when looking at the result you should see what I mean.

I think the `pad_shape` should continue to match the recommended footprint of CxD.